lib/server.ts

```diff
@@ -14,19 +15,44 @@ const jsonParser = bodyParser.json();
 const app = express();
 const httpServer = http.createServer(app);
 
-const autokitConfig = {
-    power: process.env.POWER || 'dummyPower',
-    sdMux: process.env.SD_MUX || 'dummySdMux',
-    network: process.env.NETWORK ||  'dummyNetwork',
-    video: process.env.VIDEO || 'dummyVideo',
-    usbBootPort: process.env.USB_BOOT_PORT || '4',
-    serial: process.env.SERIAL || 'dummySerial'
-}
```

Comment:
I wonder if we can keep both? 
To prevent us from maintaining state for this config, we can start with the environment variables being the primary mode of configuring your autokit instance. 
If you intend to change the configuration during runtime, then this endpoint could be used to patch the configuration. 

But, the recommended approached IMHO should be env variables that people can pop in either in the dashboard or in the docker run command using the `--env` flag. Less code to maintain, docs are clear and the entire application can have easy access to it.